my_list.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] The purpose of this project is to make you understand how innumerable work, for this reason, the use of `Array.each()` is forbidden, you should create your own `each` method without using each, kindly change this method 😄
    - Please fix this, so the next reviewer can test your `each` method correctly

Reply:
what you said it's not possible

Reply:
all my teammates they did the same thing and their project was approved

Reply:
the problem here is to perform the block with a yield statement

Reply:
It is possible, using `while` for example

Reply:
the problem is the arguments